website/index.html

```diff
@@ -178,15 +178,16 @@ <h3>Source installations</h3>
                     </li>
 
                     <li>
-                        <b>Mac OS X support:</b>
-                        <p>arbtt can run on Mac OS X systems. To compile it, you need to install the pkgconfig and pcre source packages
+                        <b>macOS support:</b>
+                        <p>arbtt can run on macOS systems. To compile it, you need to install the pkgconfig and pcre source packages
                         using <a href="http://brew.sh">Homebrew</a> or <a href="https://www.macports.org">MacPorts</a>. For Homebrew, you can execute:</p>
                         <code class="command">brew install pkgconfig</code><br>
                         <code class="command">brew install pcre</code><br>
                         <p>or for MacPorts:</p>
                         <code class="command">sudo port install pkgconfig</code><br>
                         <code class="command">sudo port install pcre</code><br>
                         <p>Then, you can compile arbtt using the usual cabal commands.</p>
+                        <code class="command">cabal configure; cabal build; cabal install</code><br>
```

Comment:
It may be misleading to present this as the single method for installing arbtt using Homebrew. The `cabal install arbtt` command mentioned above in the "From Hackage:" block also works on macOS. Here's what I do:

```sh
brew install pkg-config pcre cabal-install
cabal update
cabal install arbtt
```

I realize your change matches the existing description, so it's not at all incorrect — but if you'd like, we could use this opportunity to also mention the simpler install-from-hackage method as an option in macOS.

Reply:
The exisitng instructions are pretty-outdates, and not relaly modern. @waldyrious ’s instructions are better… would you care to put yours into a PR?

Reply:
that's what i did as well. I can update my PR to read like that. should i also remove the MacPorts instructions, or leave them up?

Reply:
I'm not a macports user so I can't comment on whether those instructions work or not. I'd rather keep them and let people raise an issue if there's a problem, than remove potentially working instructions just because. But that's just my opinion :)